ambari-agent/src/main/python/ambari_agent/Hardware.py

```diff
@@ -174,7 +174,7 @@ def osdisks(self):
        - mount path or a part of mount path is not in the blacklist
       """
       if mount["device"] not in self.IGNORE_DEVICES and\
-         mount["mountpoint"].split("/")[0] not in self.IGNORE_ROOT_MOUNTS and\
+         mount["mountpoint"].strip()[1:].split("/")[0] not in self.IGNORE_ROOT_MOUNTS and\
```

Comment:
Do we need [1:] because of the leading slash? Do we always have that slash?

Reply:
yes, it is. Valid Linux path always forms as /a/b/c, any invalid path could be ignored with no problem.

Reply:
ok, i see